app/code/community/Oyst/OneClick/controllers/Checkout/CartController.php

```diff
@@ -90,20 +90,25 @@ private function sendResponse()
     public function initOystCheckoutAction()
     {
         $params = $this->getRequest()->getParams();
-        $params['quoteId'] = Mage::getSingleton('checkout/session')->getQuoteId();
+        $params['quote_id'] = Mage::getSingleton('checkout/session')->getQuoteId();
 
-        if (!$params['quoteId']) {
+        if (!$params['quote_id']) {
             Mage::getSingleton('checkout/cart')->save();
-            $params['quoteId'] = Mage::getSingleton('checkout/session')->getQuoteId();
+            $params['quote_id'] = Mage::getSingleton('checkout/session')->getQuoteId();
+            Mage::getModel('oyst_oneclick/cart')->resetCartForSave();
         }
 
+        $quote = Mage::getSingleton('checkout/cart')->getQuote();
+        Mage::register('oyst-quote', $quote, true);
```

Comment:
le register oyst-quote est setté, mais je ne vois pas où tu l'utilises:)

Reply:
In ```Oyst_OneClick_Model_Catalog::getQuote()```

Reply:
https://github.com/oystparis/oyst-1click-magento/pull/395/files#diff-d8f0d31e3e0ef881b09dde5f8d4e2cf1R457